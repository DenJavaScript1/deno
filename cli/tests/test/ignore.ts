Deno.test({
  name: "ignore1",
  fn() {},
  ignore: true,
});

Deno.test("ignore2", function () {
}, { ignore: true });
