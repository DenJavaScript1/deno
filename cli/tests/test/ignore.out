[WILDCARD]
running 2 tests
test ignore1 ... ignored [WILDCARD]
test ignore2 ... ignored [WILDCARD]

test result: ok. 0 passed; 0 failed; 2 ignored; 0 measured; 0 filtered out [WILDCARD]
